addons/mail/static/src/core/common/thread.scss

```diff
@@ -38,3 +38,9 @@
         display: none!important;
     }
 }
+
+.o-mail-Thread-preference-hint {
+    &:hover {
+        text-decoration: underline;
+    }
+}
```

Comment:
Some other modules defined `o_underline` for that. We could just define this classname in `@mail/core.scss`, so that you just pass this classname in the proper html element in template.

That way, you don't need to define a custom classname for the element.

_(note that BEM we use is `o-mail-ComponentName-partName`, there's no split with `-` in the part name words)_